kernel/README.md

```diff
@@ -24,15 +24,91 @@ in the kernel source code. You can modify it as needed.
 After the kernel source code is ready it is possible to build the kernel.
 
 ```bash
-./build-kernel.sh build
+$ ./build-kernel.sh build
 ```
 
 
 ## Install the Kernel in the default path for Kata
 
 Kata Containers uses some default path to search a kernel to boot. To install
 on this path, the following command will install it to the default Kata
-containers path.
+containers path (`/usr/share/kata-containers/`).
+
+```bash
+$ ./build-kernel.sh install
+```
+
+## Submit Kernel Changes
+
+Kata Containers packaging repository holds the kernel configs and patches. The
+config and patches could be for many versions but we only make sure to test the
+kernel version defined in the [runtime versions file][runtime-versions-file].
```

Comment:
Why twice? [runtime versions file][runtime-versions-file].

Reply:
This is supposed to be a "named link" (tal at the bottom of this file), but it's borked so I'll comment down there...

Reply:
Links fixed